src/App.tsx

```diff
@@ -1,15 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+interface Movie {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+}
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  function matchQuery(movie: Movie) {
+    if (movie.title.toLowerCase().includes(query.toLowerCase())
+    || movie.description.toLowerCase().includes(query.toLowerCase())) {
```

Comment:
decompose it more:
- create a variable for `query.toLowerCase()` or set it in state setter with `toLowerCase()`